setup.sh

```diff
@@ -0,0 +1,8 @@
+mkdir -p ~/.streamlit/
+echo "\
+[server]\n\
+headless = true\n\
+port = $PORT\n\
+enableCORS = false\n\
+\n\
+" > ~/.streamlit/config.toml
```

Comment:
Strangely enough, for me this script generates a broken TOML file:

```
$ cat ~/.streamlit/config.toml 
[server]\nheadless = true\nport = \nenableCORS = false\n\n
```

So, just to confirm, is it intended to be used with Heroku, and is it working fine in there?

Reply:
Thanks for noting!! The config file is intended to use the Open Targets colors in the theme. It should be within `.streamlit`
Fixed in `c6f375e`